src/odm_sharing/sharing.py

```diff
@@ -21,7 +21,8 @@ def parse(schema_path: str, orgs: List[str] = []) -> OrgTableQueries:
     '''returns queries for each org and table, generated from the rules
     specified in `schema_file`
 
-    :raises OSError, ParseError:
+    :raises OSError:
```

Comment:
Can you add `:return:` directives for all these functions so that they get included in the built docs

Reply:
Can you add descriptions for the raise directives.

Reply:
Can you fill out all the missing :param directives

Reply:
done